source/codegen/templates/service_helpers.mako

```diff
@@ -305,25 +309,28 @@ ${populate_response(function_data=function_data, parameters=parameters)}\
   config = data['config']
   struct_name = common_helpers.get_data_moniker_struct_name(function_name)
   moniker_function_name = common_helpers.get_data_moniker_function_name(function_name)
-  streaming_param = common_helpers.get_streaming_parameter(function_data['parameters'])
+  streaming_param = common_helpers.get_first_streaming_parameter(function_data['parameters'])
   streaming_type = streaming_param['type']
   grpc_streaming_type = streaming_param['grpc_streaming_type']
   arg_string = service_helpers.create_args(function_data['parameters'])
   arg_string = arg_string.replace(", &moniker", "").strip()
   data_type = streaming_type.replace("[]", "")
   c_api_name = service_helpers.get_c_api_name(function_name)
   streaming_params_to_include = common_helpers.get_input_streaming_params(function_data['parameters'])
+  output_params = [p for p in function_data['parameters'] if common_helpers.is_output_parameter(p)]
+  output_params_to_define = service_helpers.get_output_streaming_param_to_define(output_params, streaming_param)
 %>\
```

Comment:
`output_params_to_define = service_helpers.get_output_streaming_param_to_define(
    [p for p in function_data['parameters'] if common_helpers.is_output_parameter(p)], 
    streaming_param
)`
will this work?

Reply:
modified as suggested